projects/Mox/module.nix

```diff
@@ -0,0 +1,84 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  options = {
+    services.mox = {
+      enable = lib.mkEnableOption "Enable Mox Mail Server";
+      configFile = lib.mkOption {
```

Comment:
This is fine at the moment, but in the future we should just make the state directory (/var/lib/mox) configurable, since it's not enough to pass a config file (try passing a config file using this option, and it will complain about missing files).